src/shared/api/constants/periods.ts

```diff
@@ -0,0 +1,6 @@
+export enum Periods {
```

Comment:
Will be used enums `Periods` and `Times` beside timer? If no, then move to timer folder